src/server/console/linux_virtual_terminal.cpp

```diff
@@ -64,11 +65,11 @@ class mir::LinuxVirtualTerminal::DeviceList
         }
 
     private:
-        DeviceListIterable(Synchronised<std::vector<Device*>>::Locked&& devices)
+        DeviceListIterable(Synchronised<std::vector<Device*>>::Locked<std::vector<Device*>>&& devices)
```

Comment:
Supporting the `Synchronised<T>::lock() const` overload makes code like this marginally less nice. We could probably fix that up with some typedefs, though.

Reply:
Yup, that's easy.

If the typedefs are at all controversial, though, I'm happy to remove that commit.